web/src/App.tsx

```diff
@@ -31,13 +30,18 @@ if (isProduction) {
   });
 }
 
+function LoadingScreen() {
+  return (
+    <main className="fixed flex h-screen w-screen flex-col pt-[56px]">
```

Comment:
I decided to leave the header out and just keep the same padding to top - I was maybe afraid that people would click the outbound links in the header while waiting (if on a slow connection), not knowing that this would take them away and require restarting any data fetching.

Reply:
Smart move.